eo-runtime/pom.xml

```diff
@@ -103,6 +109,7 @@ SOFTWARE.
               ${project.basedir}/src/test/resources/jul.properties
             </java.util.logging.config.file>
           </systemProperties>
+          <forkNode implementation="org.apache.maven.plugin.surefire.extensions.SurefireForkNodeFactory"/>
```

Comment:
@levBagryansky Why do you need to fork tests in that PR? Moreover, as I understand we still have to provide more properties in order to properly fork the test execution (like `forkCount`). Or I'm mistaken here?

Reply:
@volodya-lombrozo  Default surefire plugin corrupts native stdout. It means the text that was printed in native method will not be printed in 'mvn clean install'.  This configuration switches surefire communication channel from default to `TCP/IP` and solves the bug. 
As far as I understand, this setting does not create a new process, but changes the communication channel. 
More detailed [here](https://stackoverflow.com/questions/55272870/surefire-maven-plugin-corrupted-stdout-by-directly-writing-to-native-stream-in) and [here](https://maven.apache.org/surefire/maven-surefire-plugin/examples/process-communication.html).

Reply:
@levBagryansky Ok, I see. Thank you for the comprehensive explanation. Could I ask you add the comment to that place (to `ep-runtime/pom.xml`) as a comment?
By the way, do you print something to STDOUT in your rust code? I don't see where exactly the stream is corrupted.